resources/lib/ui/embed_extractor.py

```diff
@@ -253,7 +268,7 @@ def f(url, content):
 
 __register_extractor("http://embed.novamov.com/", __extract_swf_player)
 
-__register_extractor("https://openload.co/embed/", __extract_openload)
+#__register_extractor("https://openload.co/embed/", __extract_openload)
```

Comment:
it won't work out of the box, there is stuff to do to make it work,
just said that if you decided to comment it out (and this is the right thing to do)
then add a comment that this is temporary due to issue `#16`